src/SGFTree.cpp

```diff
@@ -290,13 +291,13 @@ int SGFTree::string_to_vertex(const std::string& movestring) const {
         return FastBoard::PASS;
     }
 
-    if (m_state.board.get_boardsize() <= 19) {
+    if (BOARD_SIZE <= 19) {
         if (movestring == "tt") {
             return FastBoard::PASS;
         }
     }
 
-    int bsize = m_state.board.get_boardsize();
+    int bsize = BOARD_SIZE;
```

Comment:
This makes no sense. This code was checking whether we correctly found a usable board size at any point.

Reply:
The removing of m_boardsize (and the corresponding getter, was done after one of seth's comments, suggesting it wasn't necessary anymore.  Should I put it back in?

Reply:
(Or should I just remove this test, since board sizes != BOARD_SIZE will throw an exception in populate_state now)